data/blog/cocoa-sdk-9.0.0.mdx

```diff
@@ -0,0 +1,54 @@
+
+![Sentry Cocoa SDK 9.0.0 Landing (Image generated by AI )](/images/cocoa-sdk-9.0.0/sdk-landing.png)
+
+## Why now?
+
+Our minimum supported OS versions were **so low** that some users were increasingly complaining about Xcode warnings caused by this — a bit too much of [Compatibility is King](https://develop.sentry.dev/sdk/philosophy/#compatibility-is-king). So it's really time to do a major update and bump the minimum supported OS versions.
+
+## Maintenance Major
+
+V9 is a maintenance major: it focuses on bumping minimum OS versions, enabling a couple of features by default, and addressing many minor API cleanup topics. It should be easy for you to upgrade. — Famous last words.
+
+## Essential Changes
+
+Here is the list of the essential changes:
+
+- We bumped the minimum supported OS versions:
+  - iOS: from 11.0 to 15.0
+  - tvOS: from 11.0 to 15.0
+  - macOS: from 10.13 to 10.14
+  - watchOS: from 4.0 to 8.0
+  - visionOS: 1.0 (unchanged)
+- We now use Xcode 16 for the precompiled XCFramework, and we set the swift-tools-version to 6.0.
+- [Structured logs](https://docs.sentry.io/platforms/apple/logs/) are no longer experimental.
+- HTTP client errors now mark sessions as errored. This provides better visibility into failed network requests in the release health data.
+- [App Hang Tracking V2](https://docs.sentry.io/platforms/apple/configuration/app-hangs/#app-hangs-v2) is now the default for everybody on iOS, tvOS, and macCatalyst.
+- [Tracing](https://docs.sentry.io/platforms/apple/tracing/): We enabled [pre-warmed app start tracing](https://docs.sentry.io/platforms/apple/tracing/instrumentation/automatic-instrumentation/#prewarmed-app-start-tracing) by default, and the app start duration now finishes when the first frame is drawn instead of when the OS posts the `UIWindowDidBecomeVisibleNotification`.
+- [Profiling](https://docs.sentry.io/platforms/apple/profiling/): we removed the deprecated [transaction-based profiling](https://docs.sentry.io/platforms/apple/profiling/#transaction-based-profiling-removed-in-900). You now have to use [UIProfiling](https://docs.sentry.io/platforms/apple/profiling/#enable-ui-profiling).
+- Compiling from source via SPM works again, but we don’t officially support it yet.
+- We dropped support for the not-very-maintained package manager [Carthage](https://github.com/Carthage/Carthage). Our support has been broken for over half a year, actually, and nobody complained. So we officially drop the support. If somebody complains, we can re-add it later.
+- We added a new SDK, `SentryDistribution`, that keeps internal builds up to date; it's the Sentry version of Emerge Tools' [ETDistribution](https://github.com/EmergeTools/ETDistribution).
+
+The complete list of all changes is long. If you're interested, check out the [migration guide](https://docs.sentry.io/platforms/apple/migration/#breaking-changes) or the [9.0.0 changelog](https://github.com/getsentry/sentry-cocoa/releases/tag/9.0.0) to see it.
```

Comment:
`m`: I would propose you add another section "How do I update?" and mention the migration guide there, maybe with some general instructions, instead of here in the same sentence as the continuation of the list. Easy to miss if someone stops reading the list half-way through.